### PR TITLE
apiVersion: graphql.gloo.solo.io/v1alpha1 Fix

### PR DESCRIPTION
**Pull request by Bslabe123**
# Description

Current version is apiVersion: graphql.gloo.solo.io/v1alpha1
Proposed version is apiVersion: graphql.gloo.solo.io/v1beta1

# Context

Users ran into this bug doing ... \ Users needed this feature to ...

# Checklist:

- [ ] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [ ] If I updated APIs (our protos) or helm values, I ran `make -B install-go-tools generated-code` to ensure there will be no code diff
- [ ] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [ ] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

BOT NOTES: 
resolves https://github.com/solo-io/solo-projects/issues/3315

### CODE REVIEW

File: changelog/v1.11.0-rc7/graphql-alpha-to-beta.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+changelog:
+  - type: FIX
+    issueLink: https://github.com/solo-io/solo-projects/issues/3315
+    description: Moves graphql api from alpha to beta
```
Comment (saiskee) [inline changelog/v1.11.0-rc7/graphql-alpha-to-beta.yaml:4]: Let's make this a bit more descriptive, these descriptions show up on our user-facing docs [here](https://docs.solo.io/gloo-edge/latest/reference/changelog/open_source/). Here's a suggested description that you can use /modify as needed:
```md
description: Moving GraphQLApi custom resource from kubernetes apiVersion `v1alpha1` to `v1beta1`
```

**Review by kdorosh — APPROVED** [review]

**Pull request merged**